src/components/common/FamilylistTable/FamilyListTable.js

```diff
@@ -53,7 +52,7 @@ const FamilyListTable = ({ family }) => {
         />
         <Column
           key="view_case"
-          render={text => <a href="/case-view">View Case</a>}
+          render={text => <a href={`/cases/:caseID`}>View Case</a>}
```

Comment:
this route name makes much more sense, will be easier to integrate with backend

Reply:
Thank you, I also plan on renaming the components for consistently within the backend in our next pull request

Reply:
I agree with this route change. Much clearer way to do it.